homeassistant/components/mqtt/alarm_control_panel.py

```diff
@@ -217,14 +230,10 @@ def state(self) -> str | None:
     @property
     def supported_features(self) -> AlarmControlPanelEntityFeature:
         """Return the list of supported features."""
-        return (
-            AlarmControlPanelEntityFeature.ARM_HOME
-            | AlarmControlPanelEntityFeature.ARM_AWAY
-            | AlarmControlPanelEntityFeature.ARM_NIGHT
-            | AlarmControlPanelEntityFeature.ARM_VACATION
-            | AlarmControlPanelEntityFeature.ARM_CUSTOM_BYPASS
-            | AlarmControlPanelEntityFeature.TRIGGER
-        )
+        supported_features = AlarmControlPanelEntityFeature(0)
+        for feature in self._config[CONF_SUPPORTED_FEATURES]:
+            supported_features |= _SUPPORTED_FEATURES[feature]
+        return supported_features
```

Comment:
May be we can assign `self._attr_supported_features` in `self._setup_from_config` and remove the property so it is not iterating every time.

Reply:
Oooh makes sense! Will do.